js_modules/dagster-ui/packages/ui-core/src/launchpad/ConfigEditorConfigPicker.tsx

```diff
@@ -304,7 +304,10 @@ const ConfigEditorPartitionPicker = React.memo((props: ConfigEditorPartitionPick
         items={partitions}
         inputProps={inputProps}
         inputValueRenderer={(partition) => partition}
-        itemPredicate={(query, partition) => query.length === 0 || partition.includes(query)}
+        itemPredicate={(query, partition) => {
+          const sanitized = query.trim().toLocaleLowerCase();
```

Comment:
Is there a reason not to use `toLocaleLowerCase` consistently everywhere? Do you know what edge cases it handles?

Reply:
<img width="1000" alt="Screenshot 2024-08-28 at 10 24 27 AM" src="https://github.com/user-attachments/assets/07e5198f-e37b-40dc-9790-36a8241010ca">
🤔